docs - update serialization docs (#28497)
metabase/metabase: docs/installation-and-operation/serialization.md

- Check the exported files into version control and audit changes to them, as the YAML files contained within the export are pretty readable.

## What gets exported

**Exports include:**

- Collections
- Dashboards
- Saved questions
- Actions
- Models
- Segments and Metrics defined in the Data Model
- SQL Snippets
- Public sharing settings for questions and dashboards
- Admin Panel settings, except for permissions
- Database connection settings
- Data Model settings

**Exports exclude:**

- Permission settings
- User accounts or settings
- Alerts on saved questions
- Personal Collections or their contents (except for the user specified with the `--user` flag; see below)
- Archived items

## Before exporting or importing

If your instance is currently running and your application database doesn't support concurrent reads and writes (like the default H2 database), you'll need to stop Metabase before exporting or importing.

## Exporting

To export the contents of a Metabase instance, use the following command in your terminal:

```
java -jar metabase.jar export [export_name] --user [example@example.com]
```

> The optional `--user` flag is used to specify a default administrator account for cases when this export is loaded into a blank Metabase instance. This user will also be marked as the creator of all artifacts that are copied over to the instance. This user's personal collection and its contents will also be included in the export. If this flag isn't specified, Metabase will assume that the instance into which you're loading already has an admin user (but the load will fail if there isn't an admin user).

Reviewer: ```suggestion
Exporting will fail if your target Metabase instance doesn't have an admin account. 

Use the optional `--user` flag  to specify a default administrator account for cases when this export is loaded into a blank Metabase instance. This user will also be marked as the creator of all artifacts that are copied over to the instance. This user's personal collection and its contents will also be included in the export.
```

Maybe move the use case to the top like this?
